packages/toml/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@covector/toml",
+  "version": "0.0.0",
+  "files": [
+    "pkg/covector_toml_bg.wasm",
+    "pkg/covector_toml.js",
+    "pkg/covector_toml.d.ts",
+    "dist/index.js",
+    "dist/index.d.ts"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "wasm-pack build --release --no-pack --target nodejs && rimraf pkg/.gitignore",
+    "postbuild": "cp ./pkg/covector_toml_bg.wasm ../action/dist && cp ./pkg/covector_toml.js ../action/dist && npm run build:ts",
```

Comment:
Does the `action` `pkg` command not end up handling this? I would have expected it to move it to the `dist` for us.

Reply:
it doesn't unfortunately